base/audit_events.go

```diff
@@ -487,12 +523,11 @@ var AuditEvents = events{
 		FilteringPermitted: false,
 		EventType:          eventTypeAdmin,
 	},
-	AuditIDDatabasePostUpgrade: {
-		Name:        "Database post-upgrade",
-		Description: "Database post-upgrade was run",
+	AuditIDPostUpgrade: {
+		Name:        "Post-upgrade",
+		Description: "Post-upgrade was run for Sync Gateway databases",
 		MandatoryFields: AuditFields{
-			"db":      "database name",
-			"preview": true,
+			"preview": false,
```

Comment:
What does the value in the MandatoryFields map represent?  Default value?

Reply:
This doesn't represent anything in fact, the type of the field is used "bool" to write an example field, and it will always show `true` in the example docs.